lib/device_impl.toit

```diff
@@ -4,6 +4,7 @@
 
 import drivers.cellular
 import device
+import encoding.json
```

Comment:
Is ubjson better?

Reply:
Probably. I will update it to ubjson. It should be no problem to change the format later because the partition should be erased across updates.